CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_iterable_source v0.10.0
+[PR #39](https://github.com/fivetran/dbt_iterable_source/pull/39) contains the following updates:
+
+## Bug Fix 
+- Removes the source freshness tests in the `src_iterable.yml`. This was originally causing run errors for users where tables weren't present in their schema but listed in the `src_iterable.yml`.
+- Removes `created_at` from the uniqueness test for `stg_iterable__event`. Uniqueness is tested solely on `unique_event_id`, a surrogate key made up of `event_id` (`_fivetran_id` in the raw table, which is a Fivetran-created unique identifier derived from hashing campaign_id, created_at, and event_name) and `_fivetran_user_id`.
```

Comment:
Great entry - suggestion to add a bit more context around the user_id field similar to what you shared with the event_id since these are both Fivetran-created and we should add more details around them.
```suggestion
- Removes `created_at` from the uniqueness test for `stg_iterable__event`. Uniqueness is tested solely on `unique_event_id`, a surrogate key made up of `event_id` (`_fivetran_id` in the raw table, which is a Fivetran-created unique identifier derived from hashing campaign_id, created_at, and event_name) and `_fivetran_user_id` (a Fivetran-created column derived from a hash of `user_id` and/or `email`).
```

Reply:
ah good suggestion! thanks